src/SearchResults/index.js

```diff
@@ -239,6 +239,7 @@ function SearchResults(state, algoliaResponse) {
 
   var facetsIndices = getIndices(state.facets);
   var disjunctiveFacetsIndices = getIndices(state.disjunctiveFacets);
+  var disjunctiveIndexResult;
```

Comment:
Set an initial value here could help read the code (we know the type of the variable).